### ISSUE TITLE
yaml: unmarshal errors

### ISSUE THREAD

**Issue by JakeMeloche** [aquasecurity/aqua-helm#581]
https://github.com/aquasecurity/aqua-helm/blob/2022.4/enforcer/templates/enforcer-daemonset.yaml#L47
https://github.com/aquasecurity/aqua-helm/blob/2022.4/enforcer/templates/enforcer-daemonset.yaml#L136

running the helm chart through Kustomize it gets yaml unmarshall errors due to the duplicate key in the above lines. Set statically in L47 and via the values.yaml file on L136

**Comment by awiesner4** [comment]
I had sent this to @JakeMeloche .   Our kustomization is:

```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization

helmCharts:
  - name: enforcer
    version: 6.5.14
    repo: https://helm.aquasec.com
    namespace: aqua
    releaseName: aqua-enforcer
    valuesFile: values.yaml
```

Where the only thing we set in the values file is `platform: "eks"`

The issue is that the `hostPID` key is duplicated in the `enforcer-daemonset.yaml` and, while Helm appears to handle this, Kustomize does not.

**Comment by awiesner4** [comment]
Just tested and changing the version to `6.2.8` allows Kustomize to render properly

**Comment by KoppulaRajender** [comment]
fixed in https://github.com/aquasecurity/aqua-helm/pull/584, released enforcer 2022.4.1 with the fix.
